project_recalculate/models/project_task.py

```diff
@@ -251,8 +253,9 @@ def task_recalculate(self):
             })
         return True
 
-    @api.one
+    @api.multi
     def write(self, vals):
-        vals = self._dates_onchange(vals)
-        vals = self._estimated_days_prepare(vals)
-        return super(ProjectTask, self).write(vals)
+        for this in self:
+            vals = this._dates_onchange(vals)
+            vals = this._estimated_days_prepare(vals)
+            return super(ProjectTask, this).write(vals)
```

Comment:
You can't return inside a loop. You should discard the returned value and return always `True`.